libs/common-types/src/lib.rs

```diff
@@ -504,6 +504,7 @@ pub enum Adjustment<Amount> {
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 pub enum FeeKey {
 	CommitAnchor,
+	PreCommitDeposit,
```

Comment:
I should have looked at this earlier, my bad, but wouldn't it make more sense to call these keys something like
```
AnchorCommitDeposit
AnchorPreCommitDeposit
```

Since there could possibly be other pre-commits in the future not related to anchors? And for consistency.

Reply:
Good idea! Even we can add a sub-enum per pallet I think. Something like:
```rust
FeeKey::Anchor(Anchor::CommitDeposit);
```